.github/workflows/release_staging.yml

```diff
@@ -48,3 +70,9 @@ jobs:
           SERVICEACCOUNT_TOKEN: ${{ secrets.SERVICEACCOUNT_TOKEN }}
           KUBE_SERVER: ${{ secrets.KUBE_SERVER }}
           DOCKERHUB_ORGANISATION: ${{ secrets.DOCKERHUB_ORGANISATION }}
+      - name: Send Slack Notification
```

Comment:
we dont need to have slack notification for staging releases, please revert the changes on the staging workflow